### DISCUSSION TITLE
Pulumi provider

### DISCUSSION THREAD

**Discussion by benjick** [coollabsio/coolify#2426] [Feature Requests]
Coolify looks cool 😎, but there's no Pulumi provider available for now

**Comment by Thijmen** [comment]
What do you want to deploy with Pulumi? Do you have some concrete examples?
To be honest; in my personal opinion, I don't think Pulumi is something for Coolify as Pulumi is mostly aimed at IaC. Coolify is more for deploying applications and automatically deploying them.

**Reply by benjick** [reply]
Maybe I've misunderstood coolify, I'll take another read!

**Reply by benjick** [reply]
I don't understand how I will handle stuff like env vars. Should I log into the web ui and update them manually?

**Reply by Thijmen** [reply]
If you want to deploy applications within Coolify, then: yes.

**Reply by benjick** [reply]
I see! Thanks for clarifying
